avocado/core/varianter.py

```diff
@@ -385,7 +385,7 @@ def _process_default_params(self, args):
         self.default_params.clear()     # We don't need these anymore
         # FIXME: Backward compatibility params, to be removed when 36 LTS is
         # discontinued
-        if (not getattr(args, "variants_skip_defaults", False) and
+        if (not getattr(args, "variants-skip-defaults", False) and
```

Comment:
I thought about it too, but this is a meta not-arg-related argument and using variable name which can't clash with args seems like a safer option.